filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilderFactory.scala

```diff
@@ -11,10 +11,13 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 
 final case class BlobPathBuilderFactory(globalConfig: Config, instanceConfig: Config) extends PathBuilderFactory {
```

Comment:
Looking at how things work elsewhere, I think that when this actually executes, the global and instance conf objects passed to this class will be the ones pulled out of `filesystems.blob`, so you'll be able to reference just the inner key names:
```
val workspaceId: String = instanceConfig.getString("workspaceId")
val workspaceManagerURL = globalConfig.getString("workspace-manager-url")
```